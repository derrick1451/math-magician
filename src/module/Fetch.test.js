/** @format */

import React from 'react';
import renderer from 'react-test-renderer';
import Fetch from './Fetch';

describe('Home page component <calculator />', () => {
  test('Snapshot of the home page', () => {
    const quotes = [
      {
        title: 'title',
        author: 'author',
        quote: 'happy coding is cool',
      },
    ];
    const tree = renderer.create(<Fetch quotes={quotes} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
