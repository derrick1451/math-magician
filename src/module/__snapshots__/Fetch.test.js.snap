// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page component <calculator /> Snapshot of the home page 1`] = `
<div
  className="list-container"
/>
`;
